Example/Example/ViewControllers/DemoViewController.swift

```diff
@@ -0,0 +1,134 @@
+import UIKit
+import SalemoveSDK
+
+class DemoViewController: UIViewController {
+    @IBOutlet weak var engagementButton: UIButton!
+    @IBOutlet weak var configurationButton: UIButton!
+
+    private var statusViewController: EngagementStatusViewController?
+
+    // MARK: System
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        Salemove.sharedInstance.pushHandler = {
+            push in
+
+            // Do any additional refresh/updates when a user interacts with the notification
+        }
+    }
+
+    // MARK: Initialisation
+
+    @IBAction func queueMessage(_ sender: Any) {
+        let interactor = EngagementStatusViewController.initStoryboardInstance()
```

Comment:
This looks almost exactly the same as in `handleQueues`. Could be extracted.

Reply:
This wasn't addressed. First 9 line in this method and `showQueueSelection` almost exactly the same.